sdk/logical/version.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+package logical;
+
+option go_package = "github.com/hashicorp/vault/sdk/logical";
+
+message Empty {}
+
+// VersionReply is the reply for the Version method.
+message VersionReply {
+  string plugin_name = 1;
+  string version = 2;
+}
+
+// Versioned is an optional RPC service implemented by plugins.
+service Versioned {
```

Comment:
WDYT about `PluginVersion` to differentiate from Vault version and protocol version?

Reply:
:+1: